### PR TITLE
Clownless Skates

### PR DESCRIPTION
**Pull request by KittenColony**
## About the PR
A pair of skates that are monochromatic, can only be found in maints lockers

## Why / Balance
Style choice since not everyone wants clown skates

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl: KittenColony & mrgreen06
add: Added gray skates, you soulless corpo.
tweak: Added Honk! to skate description

### CODE REVIEW

**Comment by KittenColony** [comment]
Green I need your fucking meta file, where did you PUT IT

File: Resources/Textures/_CD/Clothing/Shoes/Specific/grayskates.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by brainfood1183 (github), altered by discord: mrgreen06 (358779039810322433)",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon"
```
Comment (Skarletto) [inline Resources/Textures/_CD/Clothing/Shoes/Specific/grayskates.rsi/meta.json:11]: Make sure your file names are properly renamed as per the meta, so 'Icon' should be 'icon'

File: Resources/Textures/_CD/Clothing/Shoes/Specific/grayskates.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by brainfood1183 (github), altered by discord: mrgreen06 (358779039810322433)",
```
Comment (LankLTE) [inline Resources/Textures/_CD/Clothing/Shoes/Specific/grayskates.rsi/meta.json:4]: Add a link to a commit with the sprite (as it's currently used, just the current commit should be fine).

**Review by LankLTE — CHANGES_REQUESTED** [review]
Small nitpick, rest looks good.

**Review by LankLTE — APPROVED** [review]

**Pull request merged**